README.md

```diff
@@ -1,4 +1,5 @@
-### Welcome to the *Zend Framework 2.3* Release!
+![Logo](http://framework.zend.com/images/logos/ZendFramework-logo.png)
```

Comment:
> In ZF1 is an external url (same url).. it isn't in the repo

Also in [zf-web](/zendframework/zf-web) and [zf1-extras](/zendframework/zf1-extras).